Google.Cloud.Spanner.NHibernate.Tests/Google.Cloud.Spanner.NHibernate.Tests.csproj

```diff
@@ -12,6 +12,7 @@
         <PackageReference Include="Xunit.Combinatorial" Version="1.4.1" />
         <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
         <PackageReference Include="coverlet.collector" Version="1.2.0" />
+        <PackageReference Include="NHibernate" Version="5.3.10" />
```

Comment:
nit: I don't think this is necessary. The test project should get this dependency from the main NHibernate project.